charts/k10/k10/4.5.900/questions.yaml

```
@@ -0,0 +1,295 @@
+    type: password
+    label: OIDC Client ID 
+  - variable: auth.oidcAuth.clientSecret
+    description: "Client secret given by the OIDC provider for K10"
+    type: password
+    label: OIDC Client Secret
+  - variable: auth.oidcAuth.usernameClaim
+    description: "The claim to be used as the username"
+    type: string
+    label: OIDC UserName Claim
+  - variable: auth.oidcAuth.usernamePrefix
+    description: "Prefix that has to be used with the username obtained from the username claim"
+    type: string
+    label: OIDC UserName Prefix
+  - variable: auth.oidcAuth.groupClaim
+    description: "Name of a custom OpenID Connect claim for specifying user groups"
+    type: string
+    label: OIDC group Claim
+  - variable: auth.oidcAuth.groupPrefix
+    description: "All groups will be prefixed with this value to prevent conflicts"
+    type: string
+    label: OIDC group Prefix
+
+# ========================
+# External Gateway
+# ========================
+
+- variable: externalGateway.create
+  description: "Configures an external gateway for K10 API services"
+  type: boolean
+  label: Create External Gateway
+  required: false
+  group: "External Gateway"
+  show_subquestion_if: true
+  subquestions:
+  - variable: externalGateway.annotations
+    description: "Standard annotations for the services"
+    type: multiline
+    default: ""
+    label: Annotation
+  - variable: externalGateway.fqdn.name
+    description: "Domain name for the K10 API services"
+    type: string
+    label: Domain Name
+  - variable: externalGateway.fqdn.type
+    description: "Supported gateway type: `route53-mapper` or `external-dns`"
+    type: string
+    label: Gateway Type route53-mapper or external-dns
+  - variable: externalGateway.awsSSLCertARN
+    description: "ARN for the AWS ACM SSL certificate used in the K10 API server"
+    type: multiline
+    label: ARN for the AWS ACM SSL certificate
+
+# ========================
+# Storage Management
+# ========================
+
+- variable: global.persistence.storageClass
+  label: StorageClass Name
+  description: "Specifies StorageClass Name to be used for PVCs"
+  type: string
+  required: false
+  default: ""
+  group: "Storage Management"
+
+- variable: prometheus.server.persistentVolume.storageClass
+  type: string
+  label: StorageClass Name for prometheus PVC
+  description: "StorageClassName used to create Prometheus PVC. Setting this option overwrites global StorageClass value"
+  default: ""
+  required: false
+  group: "Storage Management"
+
+- variable: prometheus.server.persistentVolume.enabled
+  type: boolean
+  label: PVS for prometheus server
+  description: "If true, K10 Prometheus server will create a Persistent Volume Claim"
+  default: true
+  required: false
+  group: "Storage Management"
+
+- variable: global.persistence.enabled
+  type: boolean
+  label: Storage Enabled
+  description: "If true, K10 will use Perstistent Colume Claim"
```

```suggestion
  description: "If true, K10 will use Persistent Volume Claim"
```
typo I would correct that